tools/metrics/gauges.go

```diff
@@ -155,27 +165,56 @@ func ServiceCountGauge(c *Client, interval time.Duration) MetricReadCloser {
 		if err != nil {
 			return nil
 		}
-		counters := map[string]int{}
+		service_plans, err := c.cf.ListServicePlans()
+		if err != nil {
+			return nil
+		}
+
+		counters := map[bool]map[bool]map[string]int{
```

Comment:
It would be nice to have a comment about what this structure means. e.g. counters[ORG_IS_TRIAL][PLAN_IS_FREE][TYPE] = number of services where ORG_IS_TRIAL is true if the service is running in an org which is in trial mode, PLAN_IS_FREE is true if the service plan is free and TYPE is the service type.